dsgrid/query/query_submitter.py

```diff
@@ -630,9 +644,10 @@ def submit(
             context.metadata = metadata
             # Refer to the comment in ProjectQuerySubmitter.submit for an explanation or if
             # you add a new customization.
-            with custom_time_zone(tz.tz_name):
-                self._process_aggregations_and_save(df, context, repartition=False)
+            with custom_time_zone(tz.tz_name):  # type: ignore
```

Comment:
Just curious--is this a message for the linter?

Reply:
It suppresses unnecessary complaints from mypy and pyright.